pages/learn/deploy/release-strategy/release-policy.md

```diff
@@ -7,45 +7,40 @@ excerpt: Manage the fleet and device release policy
+
+<img src="/img/common/release-policy/releases_card_dropdown.png" width="100%" alt="Pin fleet to release from Releases card">
+<img src="/img/common/release-policy/pin_to_release_setting.png" width="100%" alt="Pin fleet to release setting">
 
-<img src="/img/common/release-policy/mNCaLF8.gif" width="100%" alt="Pin fleet to release">
+You can also pin a fleet to a release via the CLI as follows:
+- Pin to a specific release: `balena fleet pin myorg/myfleet 91165e5`
+- Track latest: `balena fleet track-latest myorg/myfleet`
 
 ## Pin device to a release
 
 As well as pinning a fleet to a specific release, you may pin individual devices to a specific release. By default, all devices track the fleet release. However, you may wish to run a different release on select devices, for example, a development device or when performing a [canary deployment][canary].
 
-To do so, from the Device dashboard, select the device menu and choose _Pin to release_.
-
-<img src="/img/common/release-policy/obsIjdo.png" width="80%" alt="Pin device to release">
-
-Selecting _Pin to release_ opens a modal window displaying a list of all successful releases from which you can select a specific release to pin the device to.
-
-<img src="/img/common/release-policy/T3Jy5l1.png" width="60%" alt="Pin to a specific device">
-
-Once the device has updated, it is shown to be on a _Pinned_ release policy on the fleet dashboard rather than following the release policy of the fleet.
-
-<img src="/img/common/release-policy/SCESUYd.png" width="100%" alt="Device release policy">
-
-You can also pin multiple devices to a target release in the fleet dashboard by selecting them and choosing the _Pin to release_ option from the _Actions_ menu. Again this action opens a modal window allowing you to choose a specific release to pin the devices to.
+There are three ways to do so from the from the Device dashboard. The dropdown from each of these methods is identical to the one you see when trying to pin the fleet to a release:
+- Via the Devices table by selecting one or more devices and opening the `Modify` menu: <img src="/img/common/release-policy/devices_table_modify.png" width="100%">
+- Via the pencil icon that appears on-hover over the `Target release` property on the Device summary page: <img src="/img/common/release-policy/devices_target_release_pencil.png" width="100%">
+- Via the Device settings page: <img src="/img/common/release-policy/device_settings.png" width="100%">
```

Comment:
```suggestion
There are several ways to achieve this from the Device dashboard. The dropdown options are identical to the ones when trying to pin the fleet to a release:

- On the Devices page, you can select one or more devices using the checkboxes present on the table. Next, open the `Modify` menu and select `Pin to release` option.

<img src="/img/common/release-policy/devices_table_modify.png" width="100%">

- Alternatively, on hovering over the `Target release` property in the Device summary page, you will see a pencil icon appear. This can be used to pin the release as well. 

<img src="/img/common/release-policy/devices_target_release_pencil.png" width="100%">

- Lastly, the settings page of a device also has the option to pin the release.

<img src="/img/common/release-policy/device_settings.png" width="100%">
```